langs/currency-per-product-for-woocommerce.pot

```diff
@@ -74,7 +74,7 @@ msgid "Currency per Product"
 msgstr ""
 
 #: includes/admin/class-alg-wc-cpp-metaboxes.php:188
-msgid "Product Currency"
+msgid "Product currency"
```

Comment:
This pot file seems outdated. There is no folder called admin inside includes folder. The file class-alg-wc-cpp-metaboxes.php is located inside includes > settings folder. 

@mokshasharmila13 This is not the correct way to update the pot file. Update it using wp i18n command - https://github.com/wp-cli/i18n-command